docs/guides/implementation-guide.mdx

```diff
@@ -351,11 +351,11 @@ You can test your validator by configuring it with a chain with the above contra
 
 ### Relayer
 
-In addition to indexing messages dispatched from the mailbox, relayers process messages on the destination chain. This requires building metadata that satisfies the message recipient's ISM verification requirements, and signing transactions that process the message on the destination mailbox.
+In addition to indexing messages dispatched from the mailbox, a relayer processes messages on the destination chain. This requires building metadata that satisfies the message recipient's ISM verification requirements, and signing transactions that process the message on the destination mailbox.
 
 #### Metadata Builders
 
-Each module type implies a different metadata format for message verification to succeed. Relayers need each module trait (eg [multisig](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/hyperlane-core/src/traits/multisig_ism.rs)) to be implemented.
+Each module type implies a different metadata format for message verification to succeed. A Relayer will need each module trait (eg [multisig](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/hyperlane-core/src/traits/multisig_ism.rs)) to be implemented.
```

Comment:
noticing sometimes relayer is considered a proper noun "Relayer" and sometimes not, not a big deal but consistency would be good

Reply:
moving discussion to discord